partiql-ast/src/main/java/org/partiql/ast/v1/Exclude.java

```diff
@@ -9,9 +10,10 @@
 /**
  * TODO docs, equals, hashcode
  */
+@Builder
```

Comment:
Discussed offline, but the generated builder lies at: `Exclude.ExcludeBuilder`. A bit redundant -- this is non-blocking, but if you'd want to look into it, I'm just pointing it out.

Reply:
Yeah, seems like I can override the builder name to just `Exclude.Builder` -- `@Builder(builderClassName = "Builder")`. Will revise